pkg/clusteragent/clusterchecks/handler.go

```diff
@@ -0,0 +1,76 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build clusterchecks
+
+package clusterchecks
+
+import (
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery"
+)
+
+const (
+	warmupPhaseDuration = 30 * time.Second
```

Comment:
What was the `warmupPhaseDuration` being used for? Could we add a comment maybe?